src/main/scala/com/databricks/labs/overwatch/utils/Structures.scala

```diff
@@ -145,6 +145,15 @@ case class SimplifiedModuleStatusReport(
 
 case class IncrementalFilter(cronField: StructField, low: Column, high: Column)
 
+case class DBUCostDetail(
+                          organization_id: String,
+                          sku: String,
+                          contract_price: Double,
+                          activeFrom: LocalDate,
+                          activeUntil: Option[LocalDate],
```

Comment:
does it make sense to use LocalDate? I would prefer to use GMT, especially for setups with multiple regions involved

Reply:
It's a fair thought and I went round and round with this in the early days. There are subtle differences in the way spark and UNIX interpret time so I had to fall back to localDate to get server time to jive with spark time. I think the right way is global TZ of UTC and use of java.sql.timestamp but either way, definitely cannot make these changes until test suite is present.